internal/cli/cmd/cluster/buildproxy.go

```diff
@@ -159,6 +160,12 @@ func (bp *BuildClusterInstance) runBuildProxy(ctx context.Context, socketPath st
 		}
 	}
 
+	workerInfo, err := os.ReadFile(injectWorkerInfoFile)
+	if err != nil {
+		return nil, err
+	}
+	injectWorkerInfo := string(workerInfo)
```

Comment:
Think it would be a bit cleaner if this receives an already parsed proto, so we can also fail early if necessary (e.g. the proto doesn't parse). And the parent is then responsible for opening the file and decoding it.